src/main/java/org/spongepowered/common/mixin/core/entity/projectile/ThrowableEntityMixin.java

```diff
@@ -47,8 +47,7 @@
     @Shadow protected LivingEntity owner;
     @Shadow protected abstract void onImpact(RayTraceResult movingObjectPosition);
 
-    @Nullable
-    public ProjectileSource projectileSource;
+    @Nullable public ProjectileSource impl$projectileSource;
```

Comment:
Shouldn't be public at all, at the most `protected` if it's being used in sub mixins.